bestbook/views/__init__.py

```diff
@@ -271,14 +288,30 @@ def get(self, cls, _id=None):
         return {cls: [v.dict(minimal=True) for v in books.core.models[cls].all()]}
 
     @rest
-    def post(self, cls, _id=None):
+    def post(self, cls, _id=None, cls2=None):
         """To prevent a service like Open Library (or anyone else) from
         making a POST on behalf of another patron, we should require
         s3 keys.
 
         Open Library has s3 keys and we can fetch s3 keys from
         bestbook during the auth stage.
         """
+        if cls == "recommendations" and _id:
```

Comment:
```suggestion
        if cls == "recommendations" and cls2:
```
Original conditional was interfering with the recommendation approvals (line 330, below).